package.json

```diff
@@ -520,7 +520,8 @@
     "@11ty/eleventy": "^0.7.1",
     "@mocha/contributors": "^1.0.3",
     "@mocha/docdash": "^2.0.0",
-    "assetgraph-builder": "^6.6.3",
+    "assetgraph-builder": "^6.9.0",
+    "autoprefixer": "^9.4.9",
```

Comment:
Why isn't this an AG-B dependency? Don't see why it should be ours...

Reply:
autoprefixer is on a quite rapid release cycle to keep up with all the browser changes. So assetgraph-builder lets you bring your own dependency so you can update the version independently of assetgraph-builder releases.